test/reporters/json.spec.js

```diff
@@ -84,6 +85,32 @@ describe('JSON reporter', function() {
     });
   });
 
+  it('should have 1 test pass', function(done) {
+    var test = new Test(testTitle, function(done) {
+      done();
+    });
+
+    test.file = testFile;
+    suite.addTest(test);
+
+    runner.run(function(failureCount) {
+      sinon.restore();
+      expect(runner, 'to satisfy', {
+        testResults: {
+          passes: [
+            {
+              title: testTitle,
+              file: testFile,
+              speed: testSpeed
```

Comment:
It's highly unlikely that `speed` would not be `fast` here, but since we haven't mocked the runner (as in `json-stream.spec.js`), it would be safer to do this:

```suggestion
              speed: /(slow|medium|fast)/
```

Also, you can remove the addition on [L14](https://github.com/mochajs/mocha/pull/4434/files#diff-1fa46f0ad13d99b90fbf14520f5b565eR14).